### PR TITLE
Improve debugging of integration tests

### PR DESCRIPTION
**Pull request by stefanhaller**
Several improvements to make debugging integration tests work better.

### CODE REVIEW

File: pkg/integration/components/runner.go [hunk 1]
```diff
@@ -138,12 +139,15 @@ func prepareTestDir(
...
 
-func buildLazygit(raceDetector bool) error {
+func buildLazygit(debug bool, raceDetector bool) error {
 	// // TODO: remove this line!
 	// // skipping this because I'm not making changes to the app code atm.
 	// return nil
 
 	args := []string{"go", "build"}
+	if debug {
+		args = append(args, "-gcflags=all=-N -l")
```
Comment (jesseduffield) [inline pkg/integration/components/runner.go:149]: Can we add an inline comment explaining what these flags mean?

**Review by jesseduffield — APPROVED** [review]
LGTM pending that one comment

File: pkg/integration/components/runner.go (hunk 1 above)
Comment (stefanhaller) [reply pkg/integration/components/runner.go:151]: Sure, see 955fc4f36ce2.

**Comment by jesseduffield** [comment]
Looks good

**Pull request merged**